src/lib/Libcmds/parse_at.c

```diff
@@ -182,7 +182,7 @@ parse_at_list(char *list, int use_count, int abs_path)
 			;
 
 		/* Drop any trailing blanks */
-		for (b = c - 1; isspace(*b); b--)
+		for (b = c - 1; b >= list_dup && isspace(*b); b--)
```

Comment:
Please add parenthesis around (b >= list_dup) as per the style guide.

Reply:
I missed your previous comment. I've included parenthesis with my recent commit.